send_webhook_payload.py

```diff
@@ -0,0 +1,73 @@
+# send_webhook_payload.py
+# This action extracts metadata from a GitHub Actions event and sends it via HTTP to a database.
+# Note: For now, this action only supports pull request events.
+# Inputs:
+# collector_endpoint: ${{ inputs.collector-endpoint }}
+# collector_token: ${{ inputs.collector-token }}
+# jira_keys: ${{ inputs.jira-keys }}
+# event: ${{ inputs.event }}
+
+# imports
+import json
+import os
+import re
+import requests
+
+# read in env vars
+collector_endpoint = os.environ['INPUT_COLLECTOR_ENDPOINT']
+collector_token = os.environ['INPUT_COLLECTOR_TOKEN']
+jira_keys = os.environ['INPUT_JIRA_KEYS']
+event = os.environ['INPUT_EVENT']
+
+# extract all JIRA tickets
+jira_tickets = []
+for key in jira_keys.split(","):
+    regex_string = key + "-\d+"
```

Comment:
I think we might want a break character to precede the key (E.g. space, comma, newline, etc)
```suggestion
    regex_string = "\b" + key + "-\d+"
```